package.json

```diff
@@ -158,7 +158,8 @@
             "d": "cd $dir && dmd $fileName && $dir$fileNameWithoutExt",
             "haskell": "runhaskell",
             "nim": "nim compile --verbosity:0 --hints:off --run",
-            "lisp": "sbcl --script"
+            "lisp": "sbcl --script",
+            "kit": "kitc --run"
```

Comment:
I think it is not needed. Since there seems no kit plugin that has language id called `kid`.

Reply:
There is, [kitlang.org](https://www.kitlang.org), just the extension isn't on the marketplace yet. In my build tool, which among other things adds the extension, I've been editing your extension to add those lines for a while now.